Cargo.toml

```diff
@@ -1,20 +1,12 @@
 [package]
 name = "eframe_template"
-default-run = "eframe_template_bin"
 version = "0.1.0"
 authors = ["Emil Ernerfeldt <emil.ernerfeldt@gmail.com>"]
 edition = "2021"
 rust-version = "1.60"
 
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
-[[bin]]
-name = "eframe_template_bin"
-path = "src/main.rs"
-
-[lib]
```

Comment:
I assume these options were to allow raw wasm compilation? that's why i removed them as we just use main.rs instead of custom start functions in lib.rs .